data/ErArvindk.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "Arvind Kumar",
+  "type": "personal",
+  "bio": " Computer Science Student | fullstack developer | DSA | A geek |learner",
+
+  "tags": [
+    "Javascript",
+    "HTML",
+    "CSS",
+    "Python",
+    "DSA",
+    "wev dev"
```

Comment:
Hi @ErArvindk, there is a minor typo in the spelling of **'web'**. Please correct it :+1: